robottelo/hosts.py

```diff
@@ -1756,6 +1778,10 @@ def enable_satellite_or_capsule_module_for_rhel8(self):
         Note: Make sure required repos are enabled before using this.
         """
         if self.os_version.major == 8:
+            if settings.server.is_ipv6:
+                self.execute(
+                    f"echo -e 'proxy={settings.server.http_proxy_ipv6_url}' >> /etc/dnf/dnf.conf"
+                )
```

Comment:
The question is out of scope of this PR. But I think it is because the modularity started in RHEL8 but the condition could be changed to `self.os_version.major == 8`, I shall leave it to teams to fix.